build_tools/release_automation.py

```diff
@@ -49,6 +49,7 @@
         {'name': 'Alina, Gervaise','affiliation': 'University of Tennessee Knoxville'},
         {'name': 'Attala, Ziggy', 'affiliation': 'STFC - Rutherford Appleton Laboratory'},
         {'name': 'Bakker, Jurrian','affiliation': 'Technical Unviersity Delft'},
+        {'name': 'Beaucage, Peter','affiliation': 'National Institute of Standards and Technology'},
```

Comment:
```suggestion
        {'name': 'Beaucage, Peter','affiliation': 'National Institute of Standards and Technology', 'orcid': '0000-0002-2147-0728'},
```

add orcid for Beaucage